proto/stride/stakeibc/tx.proto

```diff
@@ -314,3 +316,26 @@ message MsgUpdateTradeRoute {
   ];
 }
 message MsgUpdateTradeRouteResponse {}
+
+// Registers a community pool rebate by specifying the amount of the liquid
+// stake
+message MsgRegisterCommunityPoolRebate {
```

Comment:
nit: afaiu this fn is used to both register and update - should we call it `SetRegisterCommunityPoolRebate` or `UpdateCommunityPoolRebate`?

Reply:
hmm, I'm not a big fan of `SetRegister`. Update sounds fine but it's similar to Register in that it only describes one case

I think my preference is to keep it as register cause in practice we'll probably only call this once, and even updating the an rebate can be thought of as "registering a new rebate"

Reply:
Ah I meant to type `SetCommunityPoolRebate`. Slightly prefer that to `RegisterCommunityPoolRebate` but will defer to you! Please feel free to resolve once you make a decision.

Reply:
ah yeah Set sounds good! will update

Reply:
@riley-stride last open comment here - lmk if you're signed off

Reply:
looks good - signed off